stan/math/prim/arr.hpp

```diff
@@ -14,7 +14,8 @@
 #include <stan/math/prim/arr/fun/rep_array.hpp>
 #include <stan/math/prim/arr/fun/scaled_add.hpp>
 #include <stan/math/prim/arr/fun/sub.hpp>
-#include <stan/math/prim/arr/fun/sum.hpp>
+#include <stan/math/prim/arr/fun/value_of.hpp>
```

Comment:
No prob... I already fixed it.

On Thu, Mar 10, 2016 at 9:40 AM, Marco Inacio <notifications@github.com>
wrote:

> In stan/math/prim/arr.hpp
> <https://github.com/stan-dev/math/pull/244#discussion_r55687892>:
>
> > @@ -14,7 +14,8 @@
> >  #include <stan/math/prim/arr/fun/rep_array.hpp>
> >  #include <stan/math/prim/arr/fun/scaled_add.hpp>
> >  #include <stan/math/prim/arr/fun/sub.hpp>
> > -#include <stan/math/prim/arr/fun/sum.hpp>
> > +#include <stan/math/prim/arr/fun/value_of.hpp>
>
> ops, sorry, removed that line by mistake.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/stan-dev/math/pull/244/files#r55687892>.
>